gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxButton,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState(0);
+
+  function handleTabSelect(index: number) {
```

Comment:
It occurs to me that all of these `handleTabSelect` functions are redundant.  They could be gotten rid of and `setActiveTabId` used directly in their place.

Reply:
> It occurs to me that all of these `handleTabSelect` functions are redundant. They could be gotten rid of and `setActiveTabId` used directly in their place.

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/f7fe532764bc5ebd4b64a1f6baf66f8bbb6b1f99